docs/how-to-add-gpt-extensions.md

```diff
@@ -0,0 +1,199 @@
+## How to add GPT extensions to editor
```

Comment:
I suggest renaming filename by analogy with other files: `docs/how-to-connect-gpt-extensions.md`.